SwiftBasicsExercise.playground/Pages/WorkingWithText.xcplaygroundpage/Contents.swift

```diff
@@ -0,0 +1,76 @@
+//: [Previous](@previous)
+
+import Foundation
+
+//: [Next](@next)
+
+//Working with text
+//1
+var str = "Hello"
+var formatted_str = ""
+if str.count<=2{
+    formatted_str+=str
+
+}
+else{
```

Comment:
This look very weird in terms of formatting, please follow swift type guide provided to you